demos/openamp/README.md

```diff
@@ -1,66 +1,64 @@
-====================================================
-
-# the hash is only required as our input system device tree
-# has not been updated to the latest bus naming used in the
-# openamp assists
-
-% git clone https://github.com/devicetree-org/lopper.git -b systemdt-linaro-demo
-% cd lopper
-
-# ensure that the support requirements are installed
-% cat Pipfile
-
-[[source]]
-url = "https://pypi.org/simple"
-verify_ssl = true
-name = "pypi"
-
-[packages]
-flask = "*"
-flask-restful = "*"
-pandas = "*"
-"ruamel.yaml" = "*"
-anytree = "*"
-humanfriendly = "*"
-
-2) Change into the lopper demo directory
-========================================
-
-% cd demos/openamp
-
-3) execute Lopper with openamp assits and lops
-==============================================
-
-% export LOPPER_DIR="<path to your lopper clone>"
-% $LOPPER_DIR/lopper.py -f -O scratch --enhanced --permissive \
-                        -a openamp.py -a openamp_xlnx.py -a openamp-xlnx-zynq.py \
-                        -i ./inputs/openamp-overlay-zynqmp.yaml \
-                        -i $LOPPER_DIR/lopper/lops/lop-xlate-yaml.dts \
-                        -i $LOPPER_DIR/lopper/lops/lop-a53-imux.dts -i $LOPPER_DIR/lopper/lops/lop-domain-linux-a53.dts \
-                        -i $LOPPER_DIR/lopper/lops/lop-openamp-versal.dts -i $LOPPER_DIR/lopper/lops/lop-domain-linux-a53-prune.dts \
-                        inputs/system-dt/system-top.dts linux-boot.dts
-
-# The outputs from this run are: linux-boot.dts and openamp-channel-info.txt
-
-3a) linux-boot.dts
--------------------
-
-Note that this linux device tree has been created by modifying and
-transforming the input system device tree (system-top.dts), based on
-the description and values in a yaml domain file
-(openamp-overlay-zynqmp.yaml), transformed by assists (openamp,
-openampy_xlnx, openamp-xlnx-zynq) and lop files. The lop files provide
-unit transformations and control the overall flow of the
-modifications, while the assists provide more complex and context
-aware changes to the device tree.
-
-We can see that nodes such as reserved-memory have been created from
-the vring descriptions in the yaml file.
+# Lopper Demonstration
+
+## 1) Clone lopper, using the systemdt-linaro-demo branch
+
+The hash is only required as our input system device tree has not been updated to the latest bus naming used in the openamp assists.
+
+```
+    % git clone https://github.com/devicetree-org/lopper.git -b systemdt-linaro-demo
+    % cd lopper
+```
+    
+Ensure that the support requirements are installed.
+
+```
+    % cat Pipfile
+
+    [[source]]
+    url = "https://pypi.org/simple"
+    verify_ssl = true
+    name = "pypi"
+
+    [packages]
+    flask = "*"
+    flask-restful = "*"
+    pandas = "*"
+    "ruamel.yaml" = "*"
+    anytree = "*"
+    humanfriendly = "*"
+```
+
+## 2) Change into the lopper demo directory
+
+```
+    % cd demos/openamp
+```
+
+## 3) Execute Lopper with openamp assets and lops
```

Comment:
@zeddii Kindly confirm whether "assits" in the original file is correct.

Reply:
Yes, they "assist" with the conversion.  I am no speller but I think that it is OK from what I looked up.

Reply:
Sorry was away from my keyboard. @wmamills is correct, that should have been "assists"

Reply:
@wmamills @zeddii  Thank you for feedback - fixed.